qualtran/surface_code/physical_parameters.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import attr
+from attrs import frozen
+
+
+@frozen
+class PhysicalParameters:
+    """PhysicalParameters contains physical properties of a quantum computer.
+
+    Attributes:
+        t_gate: Clifford gate time.
+        t_meas: Measurement time.
+        physical_error_rate: Physical error rate.
+        reference: Source of these estimates.
+    """
+
+    t_gate = attr.ib(type=float, default=1e-6, repr=lambda x: f'{x:g}')  # 1us
+    t_meas = attr.ib(type=float, default=1e-6, repr=lambda x: f'{x:g}')  # 1us
```

Comment:
units in variable name; consider `us` as base unit.

Do we want defaults for these? I would think if you want sensible values you should use one of the provided instantiations